src/broker/cas_execute.c

```diff
@@ -856,6 +870,15 @@ ux_prepare (char *sql_stmt, int flag, char auto_commit_mode, T_NET_BUF * net_buf
       goto prepare_error;
     }
 
+  if (session->statements != NULL)
+    {
+      statement = session->statements[0];
+      if (statement != NULL)
+	{
+	  cub_ddl_log_stmt_type (statement->node_type);
+	}
+    }
+
```

Comment:
I wil propose a same statement:
```
if (session->statements && (statement = session->statements[0]))
  {
    cub_ddl_log_stmt_type (statement->node_type);
  }
```

Reply:
Thank you, i will change code as your recommend.